[0m2021.07.04 09:40:36 INFO  Started: Metals version 0.10.4 in workspace 'C:\Users\hp\Desktop\random\chotay' for client Visual Studio Code 1.57.1.[0m
[0m2021.07.04 09:40:42 INFO  time: initialize in 5.46s[0m
[0m2021.07.04 09:40:42 WARN  Build server is not auto-connectable.[0m
[0m2021.07.04 09:40:51 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
[0m2021.07.04 09:41:02 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.07.04 09:41:05 INFO  time: code lens generation in 3s[0m
[0m2021.07.04 09:41:05 INFO  time: code lens generation in 13s[0m
[0m2021.07.04 09:41:32 INFO  running 'C:\Users\hp\.jabba\jdk\adopt@1.11.0-11\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\hp\AppData\Local\Temp\metals11147377241194819133\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'[0m
[0m2021.07.04 09:41:33 ERROR [info] [launcher] getting org.scala-sbt sbt 1.5.4  (this may take some time)...[0m
[0m2021.07.04 09:45:56 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/main_2.12/1.5.4/main_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:45:56 ERROR downloading https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.14/scala-library-2.12.14.jar ...[0m
[0m2021.07.04 09:45:56 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/sbt/1.5.4/sbt-1.5.4.jar ...[0m
[0m2021.07.04 09:45:56 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/io_2.12/1.5.1/io_2.12-1.5.1.jar ...[0m
[0m2021.07.04 09:45:56 ERROR :: loading settings :: url = jar:file:/C:/Users/hp/AppData/Local/Temp/metals11147377241194819133/sbt-launch.jar!/org/apache/ivy/core/settings/ivysettings.xml[0m
[0m2021.07.04 09:45:56 ERROR :: loading settings :: url = jar:file:/C:/Users/hp/AppData/Local/Temp/metals11147377241194819133/sbt-launch.jar!/org/apache/ivy/core/settings/ivysettings.xml[0m
[0m2021.07.04 09:45:56 ERROR :: loading settings :: url = jar:file:/C:/Users/hp/AppData/Local/Temp/metals11147377241194819133/sbt-launch.jar!/org/apache/ivy/core/settings/ivysettings.xml[0m
[0m2021.07.04 09:45:56 ERROR :: loading settings :: url = jar:file:/C:/Users/hp/AppData/Local/Temp/metals11147377241194819133/sbt-launch.jar!/org/apache/ivy/core/settings/ivysettings.xml[0m
[0m2021.07.04 09:45:58 ERROR 	[SUCCESSFUL ] org.scala-sbt#sbt;1.5.4!sbt.jar (2350ms)[0m
[0m2021.07.04 09:45:58 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/logic_2.12/1.5.4/logic_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:45:58 ERROR 	[SUCCESSFUL ] org.scala-sbt#io_2.12;1.5.1!io_2.12.jar (2835ms)[0m
[0m2021.07.04 09:45:58 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/actions_2.12/1.5.4/actions_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:00 ERROR 	[SUCCESSFUL ] org.scala-sbt#logic_2.12;1.5.4!logic_2.12.jar (1955ms)[0m
[0m2021.07.04 09:46:00 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/main-settings_2.12/1.5.4/main-settings_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:01 ERROR 	[SUCCESSFUL ] org.scala-sbt#main_2.12;1.5.4!main_2.12.jar (5780ms)[0m
[0m2021.07.04 09:46:01 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/run_2.12/1.5.4/run_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:03 ERROR 	[SUCCESSFUL ] org.scala-sbt#actions_2.12;1.5.4!actions_2.12.jar (4450ms)[0m
[0m2021.07.04 09:46:03 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/command_2.12/1.5.4/command_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:03 ERROR 	[SUCCESSFUL ] org.scala-sbt#run_2.12;1.5.4!run_2.12.jar (1712ms)[0m
[0m2021.07.04 09:46:03 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/collections_2.12/1.5.4/collections_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:03 ERROR 	[SUCCESSFUL ] org.scala-lang#scala-library;2.12.14!scala-library.jar (8230ms)[0m
[0m2021.07.04 09:46:03 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/scripted-plugin_2.12/1.5.4/scripted-plugin_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:04 ERROR 	[SUCCESSFUL ] org.scala-sbt#main-settings_2.12;1.5.4!main-settings_2.12.jar (4274ms)[0m
[0m2021.07.04 09:46:04 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-lm-integration_2.12/1.5.4/zinc-lm-integration_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:04 ERROR 	[SUCCESSFUL ] org.scala-sbt#scripted-plugin_2.12;1.5.4!scripted-plugin_2.12.jar (1315ms)[0m
[0m2021.07.04 09:46:04 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/util-logging_2.12/1.5.4/util-logging_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:06 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-lm-integration_2.12;1.5.4!zinc-lm-integration_2.12.jar (1560ms)[0m
[0m2021.07.04 09:46:06 ERROR downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.3.0/scala-xml_2.12-1.3.0.jar ...[0m
[0m2021.07.04 09:46:06 ERROR 	[SUCCESSFUL ] org.scala-sbt#collections_2.12;1.5.4!collections_2.12.jar (3145ms)[0m
[0m2021.07.04 09:46:06 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/launcher-interface/1.3.2/launcher-interface-1.3.2.jar ...[0m
[0m2021.07.04 09:46:06 ERROR 	[SUCCESSFUL ] org.scala-sbt#command_2.12;1.5.4!command_2.12.jar (3845ms)[0m
[0m2021.07.04 09:46:07 ERROR downloading https://repo1.maven.org/maven2/com/github/ben-manes/caffeine/caffeine/2.8.5/caffeine-2.8.5.jar ...[0m
[0m2021.07.04 09:46:07 ERROR 	[SUCCESSFUL ] org.scala-sbt#util-logging_2.12;1.5.4!util-logging_2.12.jar (1755ms)[0m
[0m2021.07.04 09:46:07 ERROR downloading https://repo1.maven.org/maven2/io/get-coursier/lm-coursier-shaded_2.12/2.0.8/lm-coursier-shaded_2.12-2.0.8.jar ...[0m
[0m2021.07.04 09:46:09 ERROR 	[SUCCESSFUL ] org.scala-sbt#launcher-interface;1.3.2!launcher-interface.jar (2330ms)[0m
[0m2021.07.04 09:46:09 ERROR downloading https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-api/2.11.2/log4j-api-2.11.2.jar ...[0m
[0m2021.07.04 09:46:12 ERROR 	[SUCCESSFUL ] org.apache.logging.log4j#log4j-api;2.11.2!log4j-api.jar (3430ms)[0m
[0m2021.07.04 09:46:12 ERROR downloading https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-core/2.11.2/log4j-core-2.11.2.jar ...[0m
[0m2021.07.04 09:46:12 ERROR 	[SUCCESSFUL ] org.scala-lang.modules#scala-xml_2.12;1.3.0!scala-xml_2.12.jar(bundle) (6860ms)[0m
[0m2021.07.04 09:46:12 ERROR downloading https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-slf4j-impl/2.11.2/log4j-slf4j-impl-2.11.2.jar ...[0m
[0m2021.07.04 09:46:14 ERROR 	[SUCCESSFUL ] com.github.ben-manes.caffeine#caffeine;2.8.5!caffeine.jar (7370ms)[0m
[0m2021.07.04 09:46:14 ERROR 	[SUCCESSFUL ] org.apache.logging.log4j#log4j-slf4j-impl;2.11.2!log4j-slf4j-impl.jar (1510ms)[0m
[0m2021.07.04 09:46:14 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/librarymanagement-ivy_2.12/1.5.2/librarymanagement-ivy_2.12-1.5.2.jar ...[0m
[0m2021.07.04 09:46:14 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/librarymanagement-core_2.12/1.5.2/librarymanagement-core_2.12-1.5.2.jar ...[0m
[0m2021.07.04 09:46:17 ERROR 	[SUCCESSFUL ] org.apache.logging.log4j#log4j-core;2.11.2!log4j-core.jar (5091ms)[0m
[0m2021.07.04 09:46:17 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/compiler-interface/1.5.5/compiler-interface-1.5.5.jar ...[0m
[0m2021.07.04 09:46:17 ERROR 	[SUCCESSFUL ] io.get-coursier#lm-coursier-shaded_2.12;2.0.8!lm-coursier-shaded_2.12.jar (10860ms)[0m
[0m2021.07.04 09:46:17 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-compile_2.12/1.5.5/zinc-compile_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:18 ERROR 	[SUCCESSFUL ] org.scala-sbt#librarymanagement-ivy_2.12;1.5.2!librarymanagement-ivy_2.12.jar (4030ms)[0m
[0m2021.07.04 09:46:18 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/util-relation_2.12/1.5.4/util-relation_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:18 ERROR 	[SUCCESSFUL ] org.scala-sbt#compiler-interface;1.5.5!compiler-interface.jar (1420ms)[0m
[0m2021.07.04 09:46:18 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/util-position_2.12/1.5.4/util-position_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:18 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-compile_2.12;1.5.5!zinc-compile_2.12.jar (1350ms)[0m
[0m2021.07.04 09:46:18 ERROR downloading https://repo1.maven.org/maven2/com/eed3si9n/sjson-new-scalajson_2.12/0.9.1/sjson-new-scalajson_2.12-0.9.1.jar ...[0m
[0m2021.07.04 09:46:19 ERROR 	[SUCCESSFUL ] org.scala-sbt#util-relation_2.12;1.5.4!util-relation_2.12.jar (1310ms)[0m
[0m2021.07.04 09:46:19 ERROR downloading https://repo1.maven.org/maven2/org/scala-lang/scala-reflect/2.12.14/scala-reflect-2.12.14.jar ...[0m
[0m2021.07.04 09:46:19 ERROR 	[SUCCESSFUL ] org.scala-sbt#util-position_2.12;1.5.4!util-position_2.12.jar (1373ms)[0m
[0m2021.07.04 09:46:19 ERROR downloading https://repo1.maven.org/maven2/com/eed3si9n/sjson-new-core_2.12/0.9.1/sjson-new-core_2.12-0.9.1.jar ...[0m
[0m2021.07.04 09:46:19 ERROR 	[SUCCESSFUL ] org.scala-sbt#librarymanagement-core_2.12;1.5.2!librarymanagement-core_2.12.jar (5939ms)[0m
[0m2021.07.04 09:46:19 ERROR downloading https://repo1.maven.org/maven2/com/eed3si9n/shaded-jawn-parser_2.12/0.9.1/shaded-jawn-parser_2.12-0.9.1.jar ...[0m
[0m2021.07.04 09:46:21 ERROR 	[SUCCESSFUL ] com.eed3si9n#sjson-new-scalajson_2.12;0.9.1!sjson-new-scalajson_2.12.jar (1423ms)[0m
[0m2021.07.04 09:46:21 ERROR downloading https://repo1.maven.org/maven2/com/eed3si9n/shaded-scalajson_2.12/1.0.0-M4/shaded-scalajson_2.12-1.0.0-M4.jar ...[0m
[0m2021.07.04 09:46:21 ERROR 	[SUCCESSFUL ] com.eed3si9n#shaded-jawn-parser_2.12;0.9.1!shaded-jawn-parser_2.12.jar (1438ms)[0m
[0m2021.07.04 09:46:21 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/completion_2.12/1.5.4/completion_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:22 ERROR 	[SUCCESSFUL ] com.eed3si9n#shaded-scalajson_2.12;1.0.0-M4!shaded-scalajson_2.12.jar (1335ms)[0m
[0m2021.07.04 09:46:22 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/task-system_2.12/1.5.4/task-system_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:22 ERROR 	[SUCCESSFUL ] com.eed3si9n#sjson-new-core_2.12;0.9.1!sjson-new-core_2.12.jar (2212ms)[0m
[0m2021.07.04 09:46:22 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/tasks_2.12/1.5.4/tasks_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:23 ERROR 	[SUCCESSFUL ] org.scala-sbt#task-system_2.12;1.5.4!task-system_2.12.jar (1379ms)[0m
[0m2021.07.04 09:46:23 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/testing_2.12/1.5.4/testing_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:23 ERROR 	[SUCCESSFUL ] org.scala-sbt#completion_2.12;1.5.4!completion_2.12.jar (1924ms)[0m
[0m2021.07.04 09:46:23 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/util-tracking_2.12/1.5.4/util-tracking_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:23 ERROR 	[SUCCESSFUL ] org.scala-sbt#tasks_2.12;1.5.4!tasks_2.12.jar (1336ms)[0m
[0m2021.07.04 09:46:23 ERROR downloading https://repo1.maven.org/maven2/org/jline/jline-terminal/3.19.0/jline-terminal-3.19.0.jar ...[0m
[0m2021.07.04 09:46:25 ERROR 	[SUCCESSFUL ] org.scala-sbt#testing_2.12;1.5.4!testing_2.12.jar (1421ms)[0m
[0m2021.07.04 09:46:25 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-classpath_2.12/1.5.5/zinc-classpath_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:25 ERROR 	[SUCCESSFUL ] org.scala-sbt#util-tracking_2.12;1.5.4!util-tracking_2.12.jar (1421ms)[0m
[0m2021.07.04 09:46:25 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-apiinfo_2.12/1.5.5/zinc-apiinfo_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:25 ERROR 	[SUCCESSFUL ] org.jline#jline-terminal;3.19.0!jline-terminal.jar (1660ms)[0m
[0m2021.07.04 09:46:25 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc_2.12/1.5.5/zinc_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:25 ERROR 	[SUCCESSFUL ] org.scala-lang#scala-reflect;2.12.14!scala-reflect.jar (5873ms)[0m
[0m2021.07.04 09:46:25 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/util-control_2.12/1.5.4/util-control_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:26 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-classpath_2.12;1.5.5!zinc-classpath_2.12.jar (1328ms)[0m
[0m2021.07.04 09:46:26 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/jline/jline/2.14.7-sbt-a1b0ffbb8f64bb820f4f84a0c07a0c0964507493/jline-2.14.7-sbt-a1b0ffbb8f64bb820f4f84a0c07a0c0964507493.jar ...[0m
[0m2021.07.04 09:46:26 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-apiinfo_2.12;1.5.5!zinc-apiinfo_2.12.jar (1480ms)[0m
[0m2021.07.04 09:46:26 ERROR downloading https://repo1.maven.org/maven2/org/jline/jline-reader/3.19.0/jline-reader-3.19.0.jar ...[0m
[0m2021.07.04 09:46:26 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc_2.12;1.5.5!zinc_2.12.jar (1353ms)[0m
[0m2021.07.04 09:46:26 ERROR downloading https://repo1.maven.org/maven2/org/jline/jline-builtins/3.19.0/jline-builtins-3.19.0.jar ...[0m
[0m2021.07.04 09:46:26 ERROR 	[SUCCESSFUL ] org.scala-sbt#util-control_2.12;1.5.4!util-control_2.12.jar (1433ms)[0m
[0m2021.07.04 09:46:26 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/util-interface/1.5.4/util-interface-1.5.4.jar ...[0m
[0m2021.07.04 09:46:27 ERROR 	[SUCCESSFUL ] org.scala-sbt.jline#jline;2.14.7-sbt-a1b0ffbb8f64bb820f4f84a0c07a0c0964507493!jline.jar (1452ms)[0m
[0m2021.07.04 09:46:27 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/core-macros_2.12/1.5.4/core-macros_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:27 ERROR 	[SUCCESSFUL ] org.jline#jline-reader;3.19.0!jline-reader.jar (1438ms)[0m
[0m2021.07.04 09:46:27 ERROR downloading https://repo1.maven.org/maven2/org/jline/jline-terminal-jna/3.19.0/jline-terminal-jna-3.19.0.jar ...[0m
[0m2021.07.04 09:46:27 ERROR 	[SUCCESSFUL ] org.scala-sbt#util-interface;1.5.4!util-interface.jar (1272ms)[0m
[0m2021.07.04 09:46:27 ERROR downloading https://repo1.maven.org/maven2/org/jline/jline-terminal-jansi/3.19.0/jline-terminal-jansi-3.19.0.jar ...[0m
[0m2021.07.04 09:46:27 ERROR 	[SUCCESSFUL ] org.jline#jline-builtins;3.19.0!jline-builtins.jar (1722ms)[0m
[0m2021.07.04 09:46:27 ERROR downloading https://repo1.maven.org/maven2/com/lmax/disruptor/3.4.2/disruptor-3.4.2.jar ...[0m
[0m2021.07.04 09:46:29 ERROR 	[SUCCESSFUL ] org.scala-sbt#core-macros_2.12;1.5.4!core-macros_2.12.jar (1359ms)[0m
[0m2021.07.04 09:46:29 ERROR downloading https://repo1.maven.org/maven2/org/scala-lang/scala-compiler/2.12.14/scala-compiler-2.12.14.jar ...[0m
[0m2021.07.04 09:46:29 ERROR 	[SUCCESSFUL ] org.jline#jline-terminal-jna;3.19.0!jline-terminal-jna.jar (1321ms)[0m
[0m2021.07.04 09:46:29 ERROR downloading https://repo1.maven.org/maven2/org/fusesource/jansi/jansi/2.1.0/jansi-2.1.0.jar ...[0m
[0m2021.07.04 09:46:29 ERROR 	[SUCCESSFUL ] org.jline#jline-terminal-jansi;3.19.0!jline-terminal-jansi.jar (1398ms)[0m
[0m2021.07.04 09:46:29 ERROR downloading https://repo1.maven.org/maven2/com/swoval/file-tree-views/2.1.6/file-tree-views-2.1.6.jar ...[0m
[0m2021.07.04 09:46:29 ERROR 	[SUCCESSFUL ] com.lmax#disruptor;3.4.2!disruptor.jar (1312ms)[0m
[0m2021.07.04 09:46:29 ERROR downloading https://repo1.maven.org/maven2/net/java/dev/jna/jna/5.5.0/jna-5.5.0.jar ...[0m
[0m2021.07.04 09:46:31 ERROR 	[SUCCESSFUL ] com.swoval#file-tree-views;2.1.6!file-tree-views.jar (1351ms)[0m
[0m2021.07.04 09:46:31 ERROR downloading https://repo1.maven.org/maven2/net/java/dev/jna/jna-platform/5.5.0/jna-platform-5.5.0.jar ...[0m
[0m2021.07.04 09:46:31 ERROR 	[SUCCESSFUL ] org.fusesource.jansi#jansi;2.1.0!jansi.jar (2630ms)[0m
[0m2021.07.04 09:46:31 ERROR downloading https://repo1.maven.org/maven2/org/jline/jline-style/3.19.0/jline-style-3.19.0.jar ...[0m
[0m2021.07.04 09:46:35 ERROR 	[SUCCESSFUL ] net.java.dev.jna#jna;5.5.0!jna.jar (5566ms)[0m
[0m2021.07.04 09:46:35 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/util-cache_2.12/1.5.4/util-cache_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:35 ERROR 	[SUCCESSFUL ] org.jline#jline-style;3.19.0!jline-style.jar (3945ms)[0m
[0m2021.07.04 09:46:35 ERROR downloading https://repo1.maven.org/maven2/com/eed3si9n/sjson-new-murmurhash_2.12/0.9.1/sjson-new-murmurhash_2.12-0.9.1.jar ...[0m
[0m2021.07.04 09:46:38 ERROR 	[SUCCESSFUL ] org.scala-sbt#util-cache_2.12;1.5.4!util-cache_2.12.jar (2451ms)[0m
[0m2021.07.04 09:46:38 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/test-agent/1.5.4/test-agent-1.5.4.jar ...[0m
[0m2021.07.04 09:46:38 ERROR 	[SUCCESSFUL ] com.eed3si9n#sjson-new-murmurhash_2.12;0.9.1!sjson-new-murmurhash_2.12.jar (1901ms)[0m
[0m2021.07.04 09:46:38 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/test-interface/1.0/test-interface-1.0.jar ...[0m
[0m2021.07.04 09:46:39 ERROR 	[SUCCESSFUL ] org.scala-sbt#test-interface;1.0!test-interface.jar (1449ms)[0m
[0m2021.07.04 09:46:39 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/compiler-bridge_2.12/1.5.5/compiler-bridge_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:39 ERROR 	[SUCCESSFUL ] org.scala-lang#scala-compiler;2.12.14!scala-compiler.jar (10225ms)[0m
[0m2021.07.04 09:46:39 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-classfile_2.12/1.5.5/zinc-classfile_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:40 ERROR 	[SUCCESSFUL ] org.scala-sbt#test-agent;1.5.4!test-agent.jar (2665ms)[0m
[0m2021.07.04 09:46:40 ERROR downloading https://repo1.maven.org/maven2/com/jcraft/jsch/0.1.54/jsch-0.1.54.jar ...[0m
[0m2021.07.04 09:46:40 ERROR 	[SUCCESSFUL ] org.scala-sbt#compiler-bridge_2.12;1.5.5!compiler-bridge_2.12.jar (1436ms)[0m
[0m2021.07.04 09:46:40 ERROR downloading https://repo1.maven.org/maven2/com/eed3si9n/gigahorse-okhttp_2.12/0.5.0/gigahorse-okhttp_2.12-0.5.0.jar ...[0m
[0m2021.07.04 09:46:40 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-classfile_2.12;1.5.5!zinc-classfile_2.12.jar (1756ms)[0m
[0m2021.07.04 09:46:40 ERROR downloading https://repo1.maven.org/maven2/com/squareup/okhttp3/okhttp-urlconnection/3.7.0/okhttp-urlconnection-3.7.0.jar ...[0m
[0m2021.07.04 09:46:42 ERROR 	[SUCCESSFUL ] com.eed3si9n#gigahorse-okhttp_2.12;0.5.0!gigahorse-okhttp_2.12.jar (1416ms)[0m
[0m2021.07.04 09:46:42 ERROR downloading https://repo1.maven.org/maven2/com/eed3si9n/gigahorse-core_2.12/0.5.0/gigahorse-core_2.12-0.5.0.jar ...[0m
[0m2021.07.04 09:46:42 ERROR 	[SUCCESSFUL ] com.squareup.okhttp3#okhttp-urlconnection;3.7.0!okhttp-urlconnection.jar (1507ms)[0m
[0m2021.07.04 09:46:42 ERROR downloading https://repo1.maven.org/maven2/com/squareup/okhttp3/okhttp/3.14.2/okhttp-3.14.2.jar ...[0m
[0m2021.07.04 09:46:42 ERROR 	[SUCCESSFUL ] net.java.dev.jna#jna-platform;5.5.0!jna-platform.jar (11999ms)[0m
[0m2021.07.04 09:46:42 ERROR downloading https://repo1.maven.org/maven2/com/typesafe/ssl-config-core_2.12/0.4.0/ssl-config-core_2.12-0.4.0.jar ...[0m
[0m2021.07.04 09:46:43 ERROR 	[SUCCESSFUL ] com.eed3si9n#gigahorse-core_2.12;0.5.0!gigahorse-core_2.12.jar (1377ms)[0m
[0m2021.07.04 09:46:43 ERROR downloading https://repo1.maven.org/maven2/org/reactivestreams/reactive-streams/1.0.2/reactive-streams-1.0.2.jar ...[0m
[0m2021.07.04 09:46:45 ERROR 	[SUCCESSFUL ] org.reactivestreams#reactive-streams;1.0.2!reactive-streams.jar (1267ms)[0m
[0m2021.07.04 09:46:45 ERROR downloading https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.26/slf4j-api-1.7.26.jar ...[0m
[0m2021.07.04 09:46:45 ERROR 	[SUCCESSFUL ] com.typesafe#ssl-config-core_2.12;0.4.0!ssl-config-core_2.12.jar(bundle) (2380ms)[0m
[0m2021.07.04 09:46:45 ERROR downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-parser-combinators_2.12/1.1.2/scala-parser-combinators_2.12-1.1.2.jar ...[0m
[0m2021.07.04 09:46:45 ERROR 	[SUCCESSFUL ] com.squareup.okhttp3#okhttp;3.14.2!okhttp.jar (3163ms)[0m
[0m2021.07.04 09:46:45 ERROR downloading https://repo1.maven.org/maven2/com/typesafe/config/1.3.3/config-1.3.3.jar ...[0m
[0m2021.07.04 09:46:46 ERROR 	[SUCCESSFUL ] org.slf4j#slf4j-api;1.7.26!slf4j-api.jar (1538ms)[0m
[0m2021.07.04 09:46:46 ERROR downloading https://repo1.maven.org/maven2/com/squareup/okio/okio/1.17.2/okio-1.17.2.jar ...[0m
[0m2021.07.04 09:46:46 ERROR 	[SUCCESSFUL ] com.jcraft#jsch;0.1.54!jsch.jar (5978ms)[0m
[0m2021.07.04 09:46:46 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-core_2.12/1.5.5/zinc-core_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:47 ERROR 	[SUCCESSFUL ] org.scala-lang.modules#scala-parser-combinators_2.12;1.1.2!scala-parser-combinators_2.12.jar(bundle) (1924ms)[0m
[0m2021.07.04 09:46:47 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-persist_2.12/1.5.5/zinc-persist_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:47 ERROR 	[SUCCESSFUL ] com.squareup.okio#okio;1.17.2!okio.jar (1302ms)[0m
[0m2021.07.04 09:46:47 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-compile-core_2.12/1.5.5/zinc-compile-core_2.12-1.5.5.jar ...[0m
[0m2021.07.04 09:46:47 ERROR 	[SUCCESSFUL ] com.typesafe#config;1.3.3!config.jar(bundle) (2486ms)[0m
[0m2021.07.04 09:46:47 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/zinc-persist-core-assembly/1.5.5/zinc-persist-core-assembly-1.5.5.jar ...[0m
[0m2021.07.04 09:46:49 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-persist_2.12;1.5.5!zinc-persist_2.12.jar (1413ms)[0m
[0m2021.07.04 09:46:49 ERROR downloading https://repo1.maven.org/maven2/net/openhft/zero-allocation-hashing/0.10.1/zero-allocation-hashing-0.10.1.jar ...[0m
[0m2021.07.04 09:46:49 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-core_2.12;1.5.5!zinc-core_2.12.jar (2863ms)[0m
[0m2021.07.04 09:46:49 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/sbinary_2.12/0.5.1/sbinary_2.12-0.5.1.jar ...[0m
[0m2021.07.04 09:46:49 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-compile-core_2.12;1.5.5!zinc-compile-core_2.12.jar (1869ms)[0m
[0m2021.07.04 09:46:49 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/protocol_2.12/1.5.4/protocol_2.12-1.5.4.jar ...[0m
[0m2021.07.04 09:46:50 ERROR 	[SUCCESSFUL ] net.openhft#zero-allocation-hashing;0.10.1!zero-allocation-hashing.jar(bundle) (1320ms)[0m
[0m2021.07.04 09:46:50 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/template-resolver/0.1/template-resolver-0.1.jar ...[0m
[0m2021.07.04 09:46:50 ERROR 	[SUCCESSFUL ] org.scala-sbt#sbinary_2.12;0.5.1!sbinary_2.12.jar (1659ms)[0m
[0m2021.07.04 09:46:50 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/ipcsocket/ipcsocket/1.3.1/ipcsocket-1.3.1.jar ...[0m
[0m2021.07.04 09:46:51 ERROR 	[SUCCESSFUL ] org.scala-sbt#template-resolver;0.1!template-resolver.jar (1276ms)[0m
[0m2021.07.04 09:46:51 ERROR downloading https://repo1.maven.org/maven2/org/checkerframework/checker-qual/3.4.1/checker-qual-3.4.1.jar ...[0m
[0m2021.07.04 09:46:52 ERROR 	[SUCCESSFUL ] org.scala-sbt.ipcsocket#ipcsocket;1.3.1!ipcsocket.jar (1504ms)[0m
[0m2021.07.04 09:46:52 ERROR downloading https://repo1.maven.org/maven2/com/google/errorprone/error_prone_annotations/2.4.0/error_prone_annotations-2.4.0.jar ...[0m
[0m2021.07.04 09:46:52 ERROR 	[SUCCESSFUL ] org.scala-sbt#protocol_2.12;1.5.4!protocol_2.12.jar (3023ms)[0m
[0m2021.07.04 09:46:52 ERROR downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-collection-compat_2.12/2.4.2/scala-collection-compat_2.12-2.4.2.jar ...[0m
[0m2021.07.04 09:46:52 ERROR 	[SUCCESSFUL ] org.checkerframework#checker-qual;3.4.1!checker-qual.jar (1429ms)[0m
[0m2021.07.04 09:46:52 ERROR downloading https://repo1.maven.org/maven2/org/scala-sbt/ivy/ivy/2.3.0-sbt-fbc4f586aeeb1591710b14eb4f41b94880dcd745/ivy-2.3.0-sbt-fbc4f586aeeb1591710b14eb4f41b94880dcd745.jar ...[0m
[0m2021.07.04 09:46:54 ERROR 	[SUCCESSFUL ] com.google.errorprone#error_prone_annotations;2.4.0!error_prone_annotations.jar (1405ms)[0m
[0m2021.07.04 09:46:54 ERROR 	[SUCCESSFUL ] org.scala-lang.modules#scala-collection-compat_2.12;2.4.2!scala-collection-compat_2.12.jar (1683ms)[0m
[0m2021.07.04 09:46:54 ERROR 	[SUCCESSFUL ] org.scala-sbt.ivy#ivy;2.3.0-sbt-fbc4f586aeeb1591710b14eb4f41b94880dcd745!ivy.jar (1991ms)[0m
[0m2021.07.04 09:46:55 ERROR 	[SUCCESSFUL ] org.scala-sbt#zinc-persist-core-assembly;1.5.5!zinc-persist-core-assembly.jar (7324ms)[0m
[0m2021.07.04 09:46:55 ERROR :: retrieving :: org.scala-sbt#boot-app[0m
[0m2021.07.04 09:46:55 ERROR 	confs: [default][0m
[0m2021.07.04 09:46:57 ERROR 	83 artifacts copied, 0 already retrieved[0m
[0m2021.07.04 09:46:57 ERROR [info] [launcher] getting Scala 2.12.14 (for sbt)...[0m
[0m2021.07.04 09:47:16 ERROR downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6.jar ...[0m
[0m2021.07.04 09:47:16 ERROR downloading https://repo1.maven.org/maven2/jline/jline/2.14.6/jline-2.14.6.jar ...[0m
[0m2021.07.04 09:47:16 ERROR downloading https://repo1.maven.org/maven2/org/fusesource/jansi/jansi/1.12/jansi-1.12.jar ...[0m
[0m2021.07.04 09:47:18 ERROR 	[SUCCESSFUL ] org.scala-lang.modules#scala-xml_2.12;1.0.6!scala-xml_2.12.jar(bundle) (1780ms)[0m
[0m2021.07.04 09:47:19 ERROR 	[SUCCESSFUL ] jline#jline;2.14.6!jline.jar (2941ms)[0m
[0m2021.07.04 09:47:19 ERROR 	[SUCCESSFUL ] org.fusesource.jansi#jansi;1.12!jansi.jar (2949ms)[0m
[0m2021.07.04 09:47:19 ERROR :: retrieving :: org.scala-sbt#boot-scala[0m
[0m2021.07.04 09:47:19 ERROR 	confs: [default][0m
[0m2021.07.04 09:47:19 ERROR 	6 artifacts copied, 0 already retrieved[0m
[0m2021.07.04 09:47:25 INFO  [info] welcome to sbt 1.5.4 (AdoptOpenJDK Java 11.0.11)[0m
[0m2021.07.04 09:47:30 INFO  [info] loading settings for project chotay-build-build from metals.sbt ...[0m
[0m2021.07.04 09:47:31 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project\project[0m
[0m2021.07.04 09:48:22 INFO  [info] loading settings for project chotay-build from metals.sbt ...[0m
[0m2021.07.04 09:48:22 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project[0m
[0m2021.07.04 09:49:13 INFO  [success] Generated .bloop\chotay-build.json[0m
[0m2021.07.04 09:49:13 INFO  [success] Total time: 51 s, completed Jul 4, 2021, 9:49:13 AM[0m
[0m2021.07.04 09:49:18 INFO  [info] loading settings for project chotay from build.sbt ...[0m
[0m2021.07.04 09:49:18 INFO  [info] set current project to hello-world (in build file:/C:/Users/hp/Desktop/random/chotay/)[0m
[0m2021.07.04 09:49:50 INFO  [success] Generated .bloop\chotay.json[0m
[0m2021.07.04 09:49:50 INFO  [success] Generated .bloop\chotay-test.json[0m
[0m2021.07.04 09:49:50 INFO  [success] Total time: 31 s, completed Jul 4, 2021, 9:49:50 AM[0m
[0m2021.07.04 09:49:50 INFO  sbt bloopInstall exit: 0[0m
[0m2021.07.04 09:49:50 INFO  time: ran 'sbt bloopInstall' in 8m17s[0m
[0m2021.07.04 09:49:50 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 43936'...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
error: Giving up on waiting for a connection, printing embedded bloop logs:
> No server running at 127.0.0.1:8212, let's fire one...
> Resolving ch.epfl.scala:bloop-frontend_2.12:1.4.8-19-4d9f966b...
error: The launcher failed to establish a bsp connection, aborting...
[0m2021.07.04 09:50:41 ERROR Failed to connect with build server, no functionality will work.
java.lang.RuntimeException: The server did not start, got FailedToOpenBspConnection
	at bloop.launcher.LauncherMain.failPromise$1(Launcher.scala:95)
	at bloop.launcher.LauncherMain.runLauncher(Launcher.scala:122)
	at scala.meta.internal.metals.BloopServers$$anon$1.run(BloopServers.scala:159)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:829)
[0m
Jul 04, 2021 9:52:52 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 36
[0m2021.07.04 09:52:52 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\build.sbt[0m
[0m2021.07.04 09:52:54 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
Jul 04, 2021 9:53:10 AM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jul 04, 2021 9:53:22 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 55
Jul 04, 2021 9:53:44 AM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jul 04, 2021 9:53:49 AM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
[0m2021.07.04 09:53:52 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
Jul 04, 2021 9:54:16 AM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jul 04, 2021 9:54:26 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 78
Jul 04, 2021 9:54:32 AM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
[0m2021.07.04 09:54:34 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
[0m2021.07.04 09:54:58 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
[0m2021.07.04 09:55:00 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
[0m2021.07.04 09:55:09 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
[0m2021.07.04 09:57:49 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
[0m2021.07.04 09:58:08 INFO  shutting down Metals[0m
[0m2021.07.04 09:59:39 INFO  Started: Metals version 0.10.4 in workspace 'C:\Users\hp\Desktop\random\chotay' for client Visual Studio Code 1.57.1.[0m
[0m2021.07.04 09:59:48 INFO  time: initialize in 9.08s[0m
[0m2021.07.04 09:59:48 INFO  Attempting to connect to the build server...[0m
[0m2021.07.04 09:59:48 WARN  no build target for: C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala[0m
[0m2021.07.04 09:59:48 INFO  skipping build import with status 'Installed'[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 34473'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:34473...
The server is listening for incoming connections at tcp://127.0.0.1:34473...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 09:59:53 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 09:59:59 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.07.04 10:00:03 INFO  time: code lens generation in 13s[0m
[0m2021.07.04 10:00:09 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 14738'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:14738...
The server is listening for incoming connections at tcp://127.0.0.1:14738...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 10:00:10 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 10:00:21 INFO  time: Connected to build server in 32s[0m
[0m2021.07.04 10:00:21 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.04 10:00:21 INFO  time: Imported build in 0.34s[0m
[0m2021.07.04 10:00:32 INFO  time: indexed workspace in 10s[0m
[0m2021.07.04 10:00:36 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 10:01:44 INFO  time: compiled chotay in 1m7s[0m
[0m2021.07.04 10:04:13 INFO  Listening for transport dt_socket at address: 54566[0m
[0m2021.07.04 10:04:13 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 10:04:13 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 10:04:14 INFO  Starting debug proxy for [Main][0m
[0m2021.07.04 10:04:13 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54566 .[0m
[0m2021.07.04 10:04:14 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 10:04:14 INFO  Hello, World![0m
[0m2021.07.04 10:04:15 INFO  Canceling debug proxy for [Main][0m
[0m2021.07.04 10:04:28 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 10:04:29 INFO  time: compiled chotay in 1.71s[0m
[0m2021.07.04 10:04:32 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 4m 38.466s)[0m
[0m2021.07.04 10:04:34 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 4m 40.641s)[0m
[0m2021.07.04 10:04:34 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 4m 40.665s)[0m
[0m2021.07.04 10:04:36 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 4m 43.175s)[0m
[0m2021.07.04 10:04:36 INFO  Listening for transport dt_socket at address: 54587[0m
[0m2021.07.04 10:04:37 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 10:04:37 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 10:04:37 INFO  Starting debug proxy for [Main][0m
[0m2021.07.04 10:04:37 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54587 .[0m
[0m2021.07.04 10:04:37 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 10:04:37 INFO  Hello World![0m
[0m2021.07.04 10:04:37 INFO  Canceling debug proxy for [Main][0m
[0m2021.07.04 10:04:38 INFO  Listening for transport dt_socket at address: 54593[0m
[0m2021.07.04 10:04:39 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 10:04:39 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 10:04:39 INFO  Starting debug proxy for [Main][0m
[0m2021.07.04 10:04:38 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54593 .[0m
[0m2021.07.04 10:04:38 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 10:04:40 INFO  Hello World![0m
[0m2021.07.04 10:04:39 INFO  Canceling debug proxy for [Main][0m
Jul 04, 2021 10:04:40 AM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: java.net.SocketException: Socket closed
org.eclipse.lsp4j.jsonrpc.JsonRpcException: java.net.SocketException: Socket closed
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:72)
	at scala.meta.internal.metals.debug.SocketEndpoint.consume(SocketEndpoint.scala:22)
	at scala.meta.internal.metals.debug.MessageIdAdapter.consume(MessageIdAdapter.scala:43)
	at scala.meta.internal.metals.debug.ServerAdapter.send(ServerAdapter.scala:30)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleClientMessage$1(DebugProxy.scala:89)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToClient$1(DebugProxy.scala:54)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.SocketException: Socket closed
	at java.base/java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:113)
	at java.base/java.net.SocketOutputStream.write(SocketOutputStream.java:150)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:69)
	... 21 more

[0m2021.07.04 10:04:40 INFO  Listening for transport dt_socket at address: 54599[0m
[0m2021.07.04 10:04:41 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 10:04:41 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 10:04:42 INFO  Starting debug proxy for [Main][0m
[0m2021.07.04 10:04:42 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54599 .[0m
[0m2021.07.04 10:04:42 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 10:04:42 INFO  Hello World![0m
[0m2021.07.04 10:04:42 INFO  Canceling debug proxy for [Main][0m
[0m2021.07.04 10:05:48 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 5m 55.453s)[0m
[0m2021.07.04 10:05:48 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 10:05:48 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 10:05:50 WARN  1 deprecation (since 2.13.0); re-run with -deprecation for details[0m
[0m2021.07.04 10:05:50 WARN  1 deprecation (since 2.13.0); re-run with -deprecation for details[0m
[0m2021.07.04 10:05:50 INFO  time: compiled chotay in 1.45s[0m
[0m2021.07.04 10:05:55 INFO  Listening for transport dt_socket at address: 60607[0m
[0m2021.07.04 10:05:55 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 10:05:55 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 10:05:55 INFO  Starting debug proxy for [Main][0m
[0m2021.07.04 10:05:55 INFO  Trying to attach to remote debuggee VM 127.0.0.1:60607 .[0m
[0m2021.07.04 10:05:55 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 10:05:56 ERROR Error: Could not find or load main class Main[0m
[0m2021.07.04 10:05:56 ERROR Caused by: java.lang.ClassNotFoundException: Main[0m
[0m2021.07.04 10:05:55 INFO  Canceling debug proxy for [Main][0m
[0m2021.07.04 10:06:32 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 10:06:34 INFO  time: compiled chotay in 2.03s[0m
[0m2021.07.04 10:06:36 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 6m 43.273s)[0m
[0m2021.07.04 10:06:41 INFO  Listening for transport dt_socket at address: 61754[0m
[0m2021.07.04 10:06:41 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 10:06:41 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 10:06:42 INFO  Starting debug proxy for [Geeks][0m
[0m2021.07.04 10:06:42 INFO  Trying to attach to remote debuggee VM 127.0.0.1:61754 .[0m
[0m2021.07.04 10:06:42 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 10:06:42 ERROR Error: Could not find or load main class Geeks[0m
[0m2021.07.04 10:06:42 ERROR Caused by: java.lang.ClassNotFoundException: Geeks[0m
[0m2021.07.04 10:06:42 INFO  Canceling debug proxy for [Geeks][0m
Jul 04, 2021 8:31:14 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.IllegalArgumentException: 29 is not a valid line number, allowed [0..4]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 29 is not a valid line number, allowed [0..4]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 29 is not a valid line number, allowed [0..4]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:80)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

[0m2021.07.04 20:32:43 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 20:32:45 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 10h 32m 51.854s)[0m
[0m2021.07.04 20:32:45 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 20:32:45 INFO  time: compiled chotay in 0.89s[0m
[0m2021.07.04 20:33:11 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 10h 33m 18.003s)[0m
[0m2021.07.04 20:33:13 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 10h 33m 20.127s)[0m
[0m2021.07.04 20:33:13 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 10h 33m 20.131s)[0m
[0m2021.07.04 20:33:15 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 10h 33m 22.317s)[0m
[0m2021.07.04 20:33:15 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 10h 33m 22.323s)[0m
[0m2021.07.04 20:33:17 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 20:33:17 INFO  Listening for transport dt_socket at address: 60729[0m
[0m2021.07.04 20:33:17 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 20:33:17 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:33:17 INFO  Trying to attach to remote debuggee VM 127.0.0.1:60729 .[0m
[0m2021.07.04 20:33:17 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 20:33:18 INFO  Listening for transport dt_socket at address: 60740[0m
[0m2021.07.04 20:33:18 ERROR java.lang.ClassNotFoundException: com.mysql.jdbc.Driver[0m
[0m2021.07.04 20:33:18 INFO  Listening for transport dt_socket at address: 60741[0m
[0m2021.07.04 20:33:18 ERROR 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)[0m
[0m2021.07.04 20:33:18 ERROR 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)[0m
[0m2021.07.04 20:33:18 ERROR 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)[0m
[0m2021.07.04 20:33:18 ERROR 	at java.base/java.lang.Class.forName0(Native Method)[0m
[0m2021.07.04 20:33:18 ERROR 	at java.base/java.lang.Class.forName(Class.java:315)[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:17)[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:33:18 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:29)[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:18 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:18 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:33:18 ERROR Read data from io exception: java.net.SocketException: Socket closed[0m
[0m2021.07.04 20:33:18 INFO  Listening for transport dt_socket at address: 60742[0m
[0m2021.07.04 20:33:19 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 20:33:19 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 20:33:20 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 20:33:21 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 20:33:21 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:33:21 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 20:33:21 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:33:21 INFO  Trying to attach to remote debuggee VM 127.0.0.1:60741 .[0m
[0m2021.07.04 20:33:21 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 20:33:21 INFO  Trying to attach to remote debuggee VM 127.0.0.1:60740 .[0m
[0m2021.07.04 20:33:21 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 20:33:21 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 20:33:21 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:33:21 INFO  Trying to attach to remote debuggee VM 127.0.0.1:60742 .[0m
[0m2021.07.04 20:33:21 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 20:33:21 ERROR java.lang.ClassNotFoundException: com.mysql.jdbc.Driver[0m
[0m2021.07.04 20:33:21 ERROR 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)[0m
[0m2021.07.04 20:33:21 ERROR 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)[0m
[0m2021.07.04 20:33:21 ERROR 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)[0m
[0m2021.07.04 20:33:21 ERROR 	at java.base/java.lang.Class.forName0(Native Method)[0m
[0m2021.07.04 20:33:21 ERROR 	at java.base/java.lang.Class.forName(Class.java:315)[0m
[0m2021.07.04 20:33:21 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:17)[0m
[0m2021.07.04 20:33:21 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:21 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:21 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:33:21 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 20:33:21 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:29)[0m
[0m2021.07.04 20:33:21 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:21 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:33:22 ERROR java.lang.ClassNotFoundException: com.mysql.jdbc.Driver[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/java.lang.Class.forName0(Native Method)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/java.lang.Class.forName(Class.java:315)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:17)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:33:22 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:29)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:33:22 ERROR java.lang.ClassNotFoundException: com.mysql.jdbc.Driver[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/java.lang.Class.forName0(Native Method)[0m
[0m2021.07.04 20:33:22 ERROR 	at java.base/java.lang.Class.forName(Class.java:315)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:17)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:33:22 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:29)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:33:22 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:33:22 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
Jul 04, 2021 8:33:28 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\runtime\AbstractFunction0.scala
java.nio.file.AccessDeniedException: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\runtime\AbstractFunction0.scala
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:274)
	at java.base/sun.nio.fs.AbstractFileSystemProvider.deleteIfExists(AbstractFileSystemProvider.java:110)
	at java.base/java.nio.file.Files.deleteIfExists(Files.java:1181)
	at java.base/java.nio.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:117)
	at java.base/java.nio.file.Files.copy(Files.java:1298)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionAbsolutePathBuffers.toFileOnDisk(MetalsEnrichments.scala:319)
	at scala.meta.internal.metals.DestinationProvider.$anonfun$fromSymbol$3(DefinitionProvider.scala:336)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:334)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:354)
	at scala.meta.internal.metals.DefinitionProvider.fromSymbol(DefinitionProvider.scala:103)
	at scala.meta.internal.metals.StacktraceAnalyzer.findLocationForSymbol$1(StacktraceAnalyzer.scala:71)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$2(StacktraceAnalyzer.scala:76)
	at scala.PartialFunction$Unlifted.applyOrElse(PartialFunction.scala:237)
	at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:180)
	at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:167)
	at scala.collection.AbstractTraversable.collectFirst(Traversable.scala:108)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$1(StacktraceAnalyzer.scala:76)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.metals.StacktraceAnalyzer.fileLocationFromLine(StacktraceAnalyzer.scala:74)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:117)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)

Jul 04, 2021 8:33:28 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\App.scala
java.nio.file.NoSuchFileException: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\App.scala
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:96)
	at java.base/sun.nio.fs.WindowsLinkSupport.getRealPath(WindowsLinkSupport.java:257)
	at java.base/sun.nio.fs.WindowsPath.toRealPath(WindowsPath.java:891)
	at java.base/sun.nio.fs.WindowsPath.toRealPath(WindowsPath.java:42)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionAbsolutePath.dealias(CommonMtagsEnrichments.scala:370)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionString.toAbsolutePath(MetalsEnrichments.scala:481)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionString.toAbsolutePath(MetalsEnrichments.scala:475)
	at scala.meta.internal.metals.debug.DebugProtocol$.stacktraceOutputResponse(DebugProtocol.scala:73)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3(DebugProxy.scala:120)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3$adapted(DebugProxy.scala:118)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:118)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)

Jul 04, 2021 8:33:30 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.13.3-sources.jar\scala\collection\IterableOnce.scala
java.nio.file.NoSuchFileException: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.13.3-sources.jar\scala\collection\IterableOnce.scala
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:96)
	at java.base/sun.nio.fs.WindowsLinkSupport.getRealPath(WindowsLinkSupport.java:257)
	at java.base/sun.nio.fs.WindowsPath.toRealPath(WindowsPath.java:891)
	at java.base/sun.nio.fs.WindowsPath.toRealPath(WindowsPath.java:42)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionAbsolutePath.dealias(CommonMtagsEnrichments.scala:370)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionString.toAbsolutePath(MetalsEnrichments.scala:481)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionString.toAbsolutePath(MetalsEnrichments.scala:475)
	at scala.meta.internal.metals.debug.DebugProtocol$.stacktraceOutputResponse(DebugProtocol.scala:73)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3(DebugProxy.scala:120)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3$adapted(DebugProxy.scala:118)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:118)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)

Jul 04, 2021 8:33:31 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\Function0.scala
java.nio.file.AccessDeniedException: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\Function0.scala
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:274)
	at java.base/sun.nio.fs.AbstractFileSystemProvider.deleteIfExists(AbstractFileSystemProvider.java:110)
	at java.base/java.nio.file.Files.deleteIfExists(Files.java:1181)
	at java.base/java.nio.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:117)
	at java.base/java.nio.file.Files.copy(Files.java:1298)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionAbsolutePathBuffers.toFileOnDisk(MetalsEnrichments.scala:319)
	at scala.meta.internal.metals.DestinationProvider.$anonfun$fromSymbol$3(DefinitionProvider.scala:336)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:334)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:354)
	at scala.meta.internal.metals.DefinitionProvider.fromSymbol(DefinitionProvider.scala:103)
	at scala.meta.internal.metals.StacktraceAnalyzer.findLocationForSymbol$1(StacktraceAnalyzer.scala:71)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$2(StacktraceAnalyzer.scala:76)
	at scala.PartialFunction$Unlifted.applyOrElse(PartialFunction.scala:237)
	at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:180)
	at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:167)
	at scala.collection.AbstractTraversable.collectFirst(Traversable.scala:108)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$1(StacktraceAnalyzer.scala:76)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.metals.StacktraceAnalyzer.fileLocationFromLine(StacktraceAnalyzer.scala:74)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:117)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)

Jul 04, 2021 8:33:31 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\Function0.scala
java.nio.file.AccessDeniedException: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\Function0.scala
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:274)
	at java.base/sun.nio.fs.AbstractFileSystemProvider.deleteIfExists(AbstractFileSystemProvider.java:110)
	at java.base/java.nio.file.Files.deleteIfExists(Files.java:1181)
	at java.base/java.nio.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:117)
	at java.base/java.nio.file.Files.copy(Files.java:1298)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionAbsolutePathBuffers.toFileOnDisk(MetalsEnrichments.scala:319)
	at scala.meta.internal.metals.DestinationProvider.$anonfun$fromSymbol$3(DefinitionProvider.scala:336)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:334)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:354)
	at scala.meta.internal.metals.DefinitionProvider.fromSymbol(DefinitionProvider.scala:103)
	at scala.meta.internal.metals.StacktraceAnalyzer.findLocationForSymbol$1(StacktraceAnalyzer.scala:71)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$2(StacktraceAnalyzer.scala:76)
	at scala.PartialFunction$Unlifted.applyOrElse(PartialFunction.scala:237)
	at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:180)
	at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:167)
	at scala.collection.AbstractTraversable.collectFirst(Traversable.scala:108)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$1(StacktraceAnalyzer.scala:76)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.metals.StacktraceAnalyzer.fileLocationFromLine(StacktraceAnalyzer.scala:74)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:117)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)

Jul 04, 2021 8:33:31 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\Function0.scala
java.nio.file.AccessDeniedException: C:\Users\hp\Desktop\random\chotay\.metals\readonly\dependencies\scala-library-2.12.14-sources.jar\scala\Function0.scala
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:274)
	at java.base/sun.nio.fs.AbstractFileSystemProvider.deleteIfExists(AbstractFileSystemProvider.java:110)
	at java.base/java.nio.file.Files.deleteIfExists(Files.java:1181)
	at java.base/java.nio.file.CopyMoveHelper.copyToForeignTarget(CopyMoveHelper.java:117)
	at java.base/java.nio.file.Files.copy(Files.java:1298)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionAbsolutePathBuffers.toFileOnDisk(MetalsEnrichments.scala:319)
	at scala.meta.internal.metals.DestinationProvider.$anonfun$fromSymbol$3(DefinitionProvider.scala:336)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:334)
	at scala.meta.internal.metals.DestinationProvider.fromSymbol(DefinitionProvider.scala:354)
	at scala.meta.internal.metals.DefinitionProvider.fromSymbol(DefinitionProvider.scala:103)
	at scala.meta.internal.metals.StacktraceAnalyzer.findLocationForSymbol$1(StacktraceAnalyzer.scala:71)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$2(StacktraceAnalyzer.scala:76)
	at scala.PartialFunction$Unlifted.applyOrElse(PartialFunction.scala:237)
	at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:180)
	at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:167)
	at scala.collection.AbstractTraversable.collectFirst(Traversable.scala:108)
	at scala.meta.internal.metals.StacktraceAnalyzer.$anonfun$fileLocationFromLine$1(StacktraceAnalyzer.scala:76)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.metals.StacktraceAnalyzer.fileLocationFromLine(StacktraceAnalyzer.scala:74)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:117)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)

[0m2021.07.04 20:33:33 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:33:33 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:33:33 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:33:33 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:43:58 INFO  time: code lens generation in 1.01s[0m
[0m2021.07.04 20:43:58 INFO  time: code lens generation in 6.53s[0m
[0m2021.07.04 20:44:14 INFO  running 'C:\Users\hp\.jabba\jdk\adopt@1.11.0-11\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\hp\AppData\Local\Temp\metals10269551450881878019\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'[0m
[0m2021.07.04 20:44:23 INFO  [info] welcome to sbt 1.5.4 (AdoptOpenJDK Java 11.0.11)[0m
[0m2021.07.04 20:44:26 INFO  [info] loading settings for project chotay-build-build from metals.sbt ...[0m
[0m2021.07.04 20:44:28 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project\project[0m
[0m2021.07.04 20:44:30 INFO  [info] loading settings for project chotay-build from metals.sbt ...[0m
[0m2021.07.04 20:44:30 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project[0m
[0m2021.07.04 20:44:40 INFO  [success] Generated .bloop\chotay-build.json[0m
[0m2021.07.04 20:44:40 INFO  [success] Total time: 9 s, completed Jul 4, 2021, 8:44:40 PM[0m
[0m2021.07.04 20:44:46 INFO  [info] loading settings for project chotay from build.sbt ...[0m
[0m2021.07.04 20:44:46 INFO  [info] set current project to hello-world (in build file:/C:/Users/hp/Desktop/random/chotay/)[0m
[0m2021.07.04 20:46:50 INFO  [success] Generated .bloop\chotay.json[0m
[0m2021.07.04 20:46:50 INFO  [success] Generated .bloop\chotay-test.json[0m
[0m2021.07.04 20:46:50 INFO  [success] Total time: 123 s (02:03), completed Jul 4, 2021, 8:46:50 PM[0m
[0m2021.07.04 20:46:50 INFO  sbt bloopInstall exit: 0[0m
[0m2021.07.04 20:46:50 INFO  time: ran 'sbt bloopInstall' in 2m36s[0m
[0m2021.07.04 20:46:50 INFO  Disconnecting from Bloop session...[0m
[0m2021.07.04 20:46:50 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.04 20:46:50 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.04 20:46:50 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 28766'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Loading 2 projects from 'c:\Users\hp\Desktop\random\chotay\.bloop'...
[0m[32m[D][0m Loading project from 'c:\Users\hp\Desktop\random\chotay\.bloop\chotay-test.json'
[0m[32m[D][0m Loading project from 'c:\Users\hp\Desktop\random\chotay\.bloop\chotay.json'
[0m[32m[D][0m Configured SemanticDB in projects 'chotay-test', 'chotay'
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:28766...
The server is listening for incoming connections at tcp://127.0.0.1:28766...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 20:46:51 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 20:46:51 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 26585'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:26585...
The server is listening for incoming connections at tcp://127.0.0.1:26585...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 20:46:52 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 20:46:52 INFO  time: Connected to build server in 1.4s[0m
[0m2021.07.04 20:46:52 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.04 20:46:52 INFO  time: Imported build in 0.1s[0m
[0m2021.07.04 20:46:55 INFO  time: indexed workspace in 3.17s[0m
[0m2021.07.04 20:46:56 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 20:46:58 INFO  time: code lens generation in 2.39s[0m
[0m2021.07.04 20:47:00 INFO  time: compiled chotay in 4.18s[0m
[0m2021.07.04 20:47:16 INFO  skipping build import with status 'Installed'[0m
[0m2021.07.04 20:47:19 INFO  skipping build import with status 'Installed'[0m
Jul 04, 2021 8:47:21 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 186
[0m2021.07.04 20:47:49 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 58.389s)[0m
[0m2021.07.04 20:47:54 INFO  Listening for transport dt_socket at address: 58766[0m
[0m2021.07.04 20:47:54 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 20:47:54 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 20:47:55 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:47:54 INFO  Trying to attach to remote debuggee VM 127.0.0.1:58766 .[0m
[0m2021.07.04 20:47:54 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 20:48:00 ERROR com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure[0m
[0m2021.07.04 20:48:00 ERROR [0m
[0m2021.07.04 20:48:00 ERROR The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1121)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:355)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2479)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2516)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2301)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:834)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:416)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:317)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:18)[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:48:00 ERROR Caused by: java.net.ConnectException: Connection refused: connect[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.PlainSocketImpl.connect0(Native Method)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:101)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.Socket.connect(Socket.java:609)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.Socket.connect(Socket.java:558)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.Socket.<init>(Socket.java:454)[0m
[0m2021.07.04 20:48:00 ERROR 	at java.base/java.net.Socket.<init>(Socket.java:264)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:259)[0m
[0m2021.07.04 20:48:00 ERROR 	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:305)[0m
[0m2021.07.04 20:48:00 ERROR 	... 28 more[0m
[0m2021.07.04 20:48:00 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:29)[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:48:00 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:48:00 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:48:04 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
Jul 04, 2021 8:48:04 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: java.net.SocketException: Socket closed
org.eclipse.lsp4j.jsonrpc.JsonRpcException: java.net.SocketException: Socket closed
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:72)
	at scala.meta.internal.metals.debug.SocketEndpoint.consume(SocketEndpoint.scala:22)
	at scala.meta.internal.metals.debug.MessageIdAdapter.consume(MessageIdAdapter.scala:43)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3(DebugProxy.scala:121)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3$adapted(DebugProxy.scala:118)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:118)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.SocketException: Socket closed
	at java.base/java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:113)
	at java.base/java.net.SocketOutputStream.write(SocketOutputStream.java:150)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:69)
	... 26 more

[0m2021.07.04 20:52:36 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 20:52:38 INFO  time: compiled chotay in 2.03s[0m
[0m2021.07.04 20:52:40 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 5m 48.574s)[0m
[0m2021.07.04 20:52:44 INFO  Listening for transport dt_socket at address: 54428[0m
[0m2021.07.04 20:52:45 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 20:52:45 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 20:52:45 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:52:44 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54428 .[0m
[0m2021.07.04 20:52:44 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 20:52:50 ERROR com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure[0m
[0m2021.07.04 20:52:50 ERROR [0m
[0m2021.07.04 20:52:50 ERROR The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1121)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:355)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2479)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2516)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2301)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:834)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:416)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:317)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:18)[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:52:50 ERROR Caused by: java.net.ConnectException: Connection refused: connect[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.PlainSocketImpl.connect0(Native Method)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:101)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.Socket.connect(Socket.java:609)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.Socket.connect(Socket.java:558)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.Socket.<init>(Socket.java:454)[0m
[0m2021.07.04 20:52:50 ERROR 	at java.base/java.net.Socket.<init>(Socket.java:264)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:259)[0m
[0m2021.07.04 20:52:50 ERROR 	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:305)[0m
[0m2021.07.04 20:52:50 ERROR 	... 28 more[0m
[0m2021.07.04 20:52:50 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:29)[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:52:50 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:52:50 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:52:52 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
Jul 04, 2021 8:52:52 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: java.net.SocketException: Socket closed
org.eclipse.lsp4j.jsonrpc.JsonRpcException: java.net.SocketException: Socket closed
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:72)
	at scala.meta.internal.metals.debug.SocketEndpoint.consume(SocketEndpoint.scala:22)
	at scala.meta.internal.metals.debug.MessageIdAdapter.consume(MessageIdAdapter.scala:43)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3(DebugProxy.scala:121)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$3$adapted(DebugProxy.scala:118)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1(DebugProxy.scala:118)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleServerMessage$1$adapted(DebugProxy.scala:92)
	at scala.meta.internal.metals.debug.ServerAdapter.$anonfun$onReceived$1(ServerAdapter.scala:25)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.ServerAdapter.onReceived(ServerAdapter.scala:18)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToServer$1(DebugProxy.scala:58)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.SocketException: Socket closed
	at java.base/java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:113)
	at java.base/java.net.SocketOutputStream.write(SocketOutputStream.java:150)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:69)
	... 26 more

[0m2021.07.04 20:53:36 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 20:53:38 INFO  time: compiled chotay in 2.25s[0m
[0m2021.07.04 20:53:41 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 6m 50.058s)[0m
[0m2021.07.04 20:53:46 INFO  Listening for transport dt_socket at address: 54447[0m
[0m2021.07.04 20:53:47 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 20:53:47 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 20:53:47 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:53:47 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54447 .[0m
[0m2021.07.04 20:53:47 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 20:53:47 INFO  Hello[0m
[0m2021.07.04 20:53:53 ERROR com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure[0m
[0m2021.07.04 20:53:53 ERROR [0m
[0m2021.07.04 20:53:53 ERROR The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1121)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:355)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2479)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2516)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2301)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:834)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:416)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:317)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:19)[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:53:53 ERROR Caused by: java.net.ConnectException: Connection refused: connect[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.PlainSocketImpl.connect0(Native Method)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:101)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.Socket.connect(Socket.java:609)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.Socket.connect(Socket.java:558)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.Socket.<init>(Socket.java:454)[0m
[0m2021.07.04 20:53:53 ERROR 	at java.base/java.net.Socket.<init>(Socket.java:264)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:259)[0m
[0m2021.07.04 20:53:53 ERROR 	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:305)[0m
[0m2021.07.04 20:53:53 ERROR 	... 28 more[0m
[0m2021.07.04 20:53:53 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:30)[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 20:53:53 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 20:53:53 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 20:53:56 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 20:58:56 INFO  running 'C:\Users\hp\.jabba\jdk\adopt@1.11.0-11\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\hp\AppData\Local\Temp\metals16938323581277389054\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'[0m
[0m2021.07.04 20:59:01 INFO  [info] welcome to sbt 1.5.4 (AdoptOpenJDK Java 11.0.11)[0m
[0m2021.07.04 20:59:05 INFO  [info] loading settings for project chotay-build-build from metals.sbt ...[0m
[0m2021.07.04 20:59:08 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project\project[0m
[0m2021.07.04 20:59:12 INFO  [info] loading settings for project chotay-build from metals.sbt ...[0m
[0m2021.07.04 20:59:12 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project[0m
[0m2021.07.04 20:59:20 INFO  [success] Generated .bloop\chotay-build.json[0m
[0m2021.07.04 20:59:20 INFO  [success] Total time: 8 s, completed Jul 4, 2021, 8:59:21 PM[0m
[0m2021.07.04 20:59:27 INFO  [info] loading settings for project chotay from build.sbt ...[0m
[0m2021.07.04 20:59:27 INFO  [info] set current project to hello-world (in build file:/C:/Users/hp/Desktop/random/chotay/)[0m
[0m2021.07.04 21:00:43 INFO  user cancelled sbt bloopInstall[0m
[0m2021.07.04 21:00:43 INFO  time: ran 'sbt bloopInstall' in 1m46s[0m
[0m2021.07.04 21:00:43 INFO  sbt bloopInstall exit: 2147483647[0m
Jul 04, 2021 9:00:45 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 332
[0m2021.07.04 21:00:51 INFO  running 'C:\Users\hp\.jabba\jdk\adopt@1.11.0-11\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\hp\AppData\Local\Temp\metals15041155570774976091\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'[0m
Jul 04, 2021 9:00:52 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 349
[0m2021.07.04 21:00:57 INFO  [info] welcome to sbt 1.5.4 (AdoptOpenJDK Java 11.0.11)[0m
[0m2021.07.04 21:00:59 INFO  [info] loading settings for project chotay-build-build from metals.sbt ...[0m
[0m2021.07.04 21:01:01 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project\project[0m
[0m2021.07.04 21:01:05 INFO  [info] loading settings for project chotay-build from metals.sbt ...[0m
[0m2021.07.04 21:01:05 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project[0m
[0m2021.07.04 21:01:09 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:01:16 INFO  time: compiled chotay in 6.82s[0m
[0m2021.07.04 21:01:18 INFO  [success] Generated .bloop\chotay-build.json[0m
[0m2021.07.04 21:01:18 INFO  [success] Total time: 12 s, completed Jul 4, 2021, 9:01:18 PM[0m
[0m2021.07.04 21:01:25 INFO  [info] loading settings for project chotay from build.sbt ...[0m
[0m2021.07.04 21:01:25 INFO  [info] set current project to hello-world (in build file:/C:/Users/hp/Desktop/random/chotay/)[0m
[0m2021.07.04 21:01:27 INFO  [success] Generated .bloop\chotay.json[0m
[0m2021.07.04 21:01:27 INFO  [success] Generated .bloop\chotay-test.json[0m
[0m2021.07.04 21:01:27 INFO  [success] Total time: 1 s, completed Jul 4, 2021, 9:01:27 PM[0m
[0m2021.07.04 21:01:27 INFO  sbt bloopInstall exit: 0[0m
[0m2021.07.04 21:01:27 INFO  time: ran 'sbt bloopInstall' in 36s[0m
[0m2021.07.04 21:01:27 INFO  Disconnecting from Bloop session...[0m
[0m2021.07.04 21:01:27 INFO  Shut down connection with build server.[0m
No more data in the server stdin, exiting...
No more data in the server stdin, exiting...
No more data in the client stdin, exiting...
[0m2021.07.04 21:01:27 INFO  Shut down connection with build server.[0m
No more data in the server stdin, exiting...
No more data in the server stdin, exiting...
No more data in the client stdin, exiting...
[0m2021.07.04 21:01:27 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 40383'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:40383...
The server is listening for incoming connections at tcp://127.0.0.1:40383...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 21:01:28 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 21:01:28 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 28813'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:28813...
The server is listening for incoming connections at tcp://127.0.0.1:28813...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 21:01:28 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 21:01:28 INFO  time: Connected to build server in 1.5s[0m
[0m2021.07.04 21:01:28 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.04 21:01:28 INFO  time: Imported build in 0.16s[0m
[0m2021.07.04 21:01:32 INFO  time: indexed workspace in 3.27s[0m
[0m2021.07.04 21:01:39 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 11.773s)[0m
[0m2021.07.04 21:01:44 INFO  Listening for transport dt_socket at address: 64445[0m
[0m2021.07.04 21:01:44 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 21:01:44 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 21:01:45 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:01:44 INFO  Trying to attach to remote debuggee VM 127.0.0.1:64445 .[0m
[0m2021.07.04 21:01:44 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 21:01:45 INFO  Hello[0m
[0m2021.07.04 21:01:45 ERROR java.sql.SQLException: Unable to load authentication plugin 'caching_sha2_password'.[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:927)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:924)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1725)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1250)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2483)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2516)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2301)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:834)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)[0m
[0m2021.07.04 21:01:45 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 21:01:45 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 21:01:45 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 21:01:45 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:416)[0m
[0m2021.07.04 21:01:45 ERROR 	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:317)[0m
[0m2021.07.04 21:01:45 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)[0m
[0m2021.07.04 21:01:45 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:19)[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:01:45 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:30)[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:01:45 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:01:45 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:01:48 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:02:56 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:02:58 INFO  time: compiled chotay in 1.69s[0m
[0m2021.07.04 21:03:18 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:03:19 INFO  time: compiled chotay in 1.55s[0m
[0m2021.07.04 21:03:21 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 1m 53.673s)[0m
[0m2021.07.04 21:03:26 INFO  Listening for transport dt_socket at address: 64468[0m
[0m2021.07.04 21:03:26 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 21:03:26 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 21:03:27 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:03:26 INFO  Trying to attach to remote debuggee VM 127.0.0.1:64468 .[0m
[0m2021.07.04 21:03:26 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 21:03:27 INFO  Hello[0m
[0m2021.07.04 21:03:27 ERROR java.sql.SQLException: Unable to load authentication plugin 'caching_sha2_password'.[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:927)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:924)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1725)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1250)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2483)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2516)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2301)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:834)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)[0m
[0m2021.07.04 21:03:27 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 21:03:27 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 21:03:27 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 21:03:27 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:416)[0m
[0m2021.07.04 21:03:27 ERROR 	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:317)[0m
[0m2021.07.04 21:03:27 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)[0m
[0m2021.07.04 21:03:27 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:19)[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:03:27 INFO  Hello2[0m
[0m2021.07.04 21:03:27 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:32)[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:03:27 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:03:27 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:03:30 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:14:07 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:14:09 INFO  time: compiled chotay in 1.61s[0m
[0m2021.07.04 21:14:46 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:14:47 INFO  time: compiled chotay in 1.31s[0m
[0m2021.07.04 21:15:17 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:15:19 INFO  time: compiled chotay in 1.71s[0m
[0m2021.07.04 21:15:21 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 13m 52.969s)[0m
[0m2021.07.04 21:15:25 INFO  Listening for transport dt_socket at address: 61266[0m
[0m2021.07.04 21:15:26 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 21:15:26 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 21:15:26 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:15:25 INFO  Trying to attach to remote debuggee VM 127.0.0.1:61266 .[0m
[0m2021.07.04 21:15:25 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 21:15:27 INFO  Hello[0m
[0m2021.07.04 21:15:27 INFO  Hellooooooo[0m
[0m2021.07.04 21:15:27 ERROR java.sql.SQLException: Unable to load authentication plugin 'caching_sha2_password'.[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:927)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:924)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1725)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1250)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2483)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2516)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2301)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:834)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)[0m
[0m2021.07.04 21:15:27 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 21:15:27 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 21:15:27 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 21:15:27 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:416)[0m
[0m2021.07.04 21:15:27 ERROR 	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:317)[0m
[0m2021.07.04 21:15:27 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)[0m
[0m2021.07.04 21:15:27 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:20)[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:15:27 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:33)[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:15:27 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:15:27 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:15:27 INFO  Hello2[0m
[0m2021.07.04 21:15:29 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:19:42 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 18m 14.543s)[0m
[0m2021.07.04 21:19:47 INFO  Listening for transport dt_socket at address: 61329[0m
[0m2021.07.04 21:19:47 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 21:19:47 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 21:19:48 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:19:47 INFO  Trying to attach to remote debuggee VM 127.0.0.1:61329 .[0m
[0m2021.07.04 21:19:47 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 21:19:48 INFO  Hello[0m
[0m2021.07.04 21:19:48 INFO  Hellooooooo[0m
[0m2021.07.04 21:19:48 ERROR java.sql.SQLException: java.lang.ClassCastException: class java.math.BigInteger cannot be cast to class java.lang.Long (java.math.BigInteger and java.lang.Long are in module java.base of loader 'bootstrap')[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1078)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:989)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:975)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:920)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.ConnectionImpl.buildCollationMapping(ConnectionImpl.java:1074)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:3590)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2531)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2301)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:834)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)[0m
[0m2021.07.04 21:19:48 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m2021.07.04 21:19:48 ERROR 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m2021.07.04 21:19:48 ERROR 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m2021.07.04 21:19:48 ERROR 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:416)[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:317)[0m
[0m2021.07.04 21:19:48 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)[0m
[0m2021.07.04 21:19:48 ERROR 	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:20)[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:19:48 ERROR Caused by: java.lang.ClassCastException: class java.math.BigInteger cannot be cast to class java.lang.Long (java.math.BigInteger and java.lang.Long are in module java.base of loader 'bootstrap')[0m
[0m2021.07.04 21:19:48 ERROR 	at com.mysql.jdbc.ConnectionImpl.buildCollationMapping(ConnectionImpl.java:1019)[0m
[0m2021.07.04 21:19:48 ERROR 	... 28 more[0m
[0m2021.07.04 21:19:48 ERROR Exception in thread "main" java.lang.NullPointerException[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect$.delayedEndpoint$tests$ScalaJdbcConnectSelect$1(Main.scala:33)[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect$delayedInit$body.apply(Main.scala:9)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.Function0.apply$mcV$sp(Function0.scala:39)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.$anonfun$main$1(App.scala:73)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.$anonfun$main$1$adapted(App.scala:73)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.collection.AbstractIterable.foreach(Iterable.scala:920)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.main(App.scala:73)[0m
[0m2021.07.04 21:19:48 ERROR 	at scala.App.main$(App.scala:71)[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect$.main(Main.scala:9)[0m
[0m2021.07.04 21:19:48 ERROR 	at tests.ScalaJdbcConnectSelect.main(Main.scala)[0m
[0m2021.07.04 21:19:48 INFO  Hello2[0m
[0m2021.07.04 21:19:50 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
Jul 04, 2021 9:24:19 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 722
[0m2021.07.04 21:24:54 INFO  running 'C:\Users\hp\.jabba\jdk\adopt@1.11.0-11\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\hp\AppData\Local\Temp\metals18231858973632663641\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'[0m
[0m2021.07.04 21:24:59 INFO  [info] welcome to sbt 1.5.4 (AdoptOpenJDK Java 11.0.11)[0m
[0m2021.07.04 21:25:01 INFO  [info] loading settings for project chotay-build-build from metals.sbt ...[0m
[0m2021.07.04 21:25:04 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project\project[0m
[0m2021.07.04 21:25:06 INFO  [info] loading settings for project chotay-build from metals.sbt ...[0m
[0m2021.07.04 21:25:06 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project[0m
[0m2021.07.04 21:25:14 INFO  [success] Generated .bloop\chotay-build.json[0m
[0m2021.07.04 21:25:14 INFO  [success] Total time: 7 s, completed Jul 4, 2021, 9:25:15 PM[0m
[0m2021.07.04 21:25:20 INFO  [info] loading settings for project chotay from build.sbt ...[0m
[0m2021.07.04 21:25:20 INFO  [info] set current project to hello-world (in build file:/C:/Users/hp/Desktop/random/chotay/)[0m
[0m2021.07.04 21:25:34 INFO  [success] Generated .bloop\chotay-test.json[0m
[0m2021.07.04 21:25:34 INFO  [success] Generated .bloop\chotay.json[0m
[0m2021.07.04 21:25:34 INFO  [success] Total time: 13 s, completed Jul 4, 2021, 9:25:34 PM[0m
[0m2021.07.04 21:25:34 INFO  sbt bloopInstall exit: 0[0m
[0m2021.07.04 21:25:34 INFO  time: ran 'sbt bloopInstall' in 39s[0m
[0m2021.07.04 21:25:34 INFO  Disconnecting from Bloop session...[0m
[0m2021.07.04 21:25:34 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.04 21:25:34 INFO  Shut down connection with build server.[0m
No more data in the server stdin, exiting...
No more data in the server stdin, exiting...
No more data in the client stdin, exiting...
[0m2021.07.04 21:25:34 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 41506'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Loading 2 projects from 'c:\Users\hp\Desktop\random\chotay\.bloop'...
[0m[32m[D][0m Loading project from 'c:\Users\hp\Desktop\random\chotay\.bloop\chotay.json'
[0m[32m[D][0m Loading project from 'c:\Users\hp\Desktop\random\chotay\.bloop\chotay-test.json'
[0m[32m[D][0m Configured SemanticDB in projects 'chotay', 'chotay-test'
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:41506...
The server is listening for incoming connections at tcp://127.0.0.1:41506...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 21:25:34 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 21:25:34 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 27418'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:27418...
The server is listening for incoming connections at tcp://127.0.0.1:27418...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 21:25:36 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 21:25:36 INFO  time: Connected to build server in 1.61s[0m
[0m2021.07.04 21:25:36 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.04 21:25:36 INFO  time: Imported build in 0.37s[0m
Jul 04, 2021 9:25:39 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 748
[0m2021.07.04 21:25:40 INFO  time: indexed workspace in 3.71s[0m
[0m2021.07.04 21:25:58 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 23.326s)[0m
[0m2021.07.04 21:26:03 INFO  Listening for transport dt_socket at address: 49735[0m
[0m2021.07.04 21:26:03 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 21:26:03 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 21:26:04 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:26:03 INFO  Trying to attach to remote debuggee VM 127.0.0.1:49735 .[0m
[0m2021.07.04 21:26:03 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 21:26:04 INFO  Hello[0m
[0m2021.07.04 21:26:04 INFO  Hellooooooo[0m
[0m2021.07.04 21:26:04 ERROR Sun Jul 04 21:26:05 PKT 2021 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.[0m
[0m2021.07.04 21:26:06 INFO  Hello1[0m
[0m2021.07.04 21:26:06 INFO  host = 1, user = Moosa[0m
[0m2021.07.04 21:26:06 INFO  host = 2, user = Khilat[0m
[0m2021.07.04 21:26:06 INFO  host = 3, user = Shehroz[0m
[0m2021.07.04 21:26:06 INFO  host = 4, user = Asil[0m
[0m2021.07.04 21:26:06 INFO  host = 5, user = Numan[0m
[0m2021.07.04 21:26:06 INFO  host = 6, user = Jalil[0m
[0m2021.07.04 21:26:06 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:27:15 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:27:17 INFO  time: compiled chotay in 1.46s[0m
[0m2021.07.04 21:27:25 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:27:27 INFO  time: compiled chotay in 1.84s[0m
[0m2021.07.04 21:32:37 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:32:38 INFO  time: compiled chotay in 1.58s[0m
[0m2021.07.04 21:32:49 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:32:51 INFO  time: compiled chotay in 1.4s[0m
[0m2021.07.04 21:32:58 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:33:00 INFO  time: compiled chotay in 1.64s[0m
[0m2021.07.04 21:33:03 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:33:05 INFO  time: compiled chotay in 1.97s[0m
[0m2021.07.04 21:33:11 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 7m 36.231s)[0m
[0m2021.07.04 21:33:16 INFO  Listening for transport dt_socket at address: 60284[0m
[0m2021.07.04 21:33:16 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 21:33:16 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 21:33:17 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:33:16 INFO  Trying to attach to remote debuggee VM 127.0.0.1:60284 .[0m
[0m2021.07.04 21:33:16 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 21:33:18 ERROR Sun Jul 04 21:33:18 PKT 2021 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.[0m
[0m2021.07.04 21:35:27 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:35:29 INFO  time: compiled chotay in 1.67s[0m
[0m2021.07.04 21:35:34 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:35:35 INFO  time: compiled chotay in 1.26s[0m
[0m2021.07.04 21:36:45 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:39:36 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:39:37 INFO  time: compiled chotay in 1.43s[0m
[0m2021.07.04 21:43:28 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:43:29 INFO  time: compiled chotay in 1.41s[0m
[0m2021.07.04 21:43:39 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:43:41 INFO  time: compiled chotay in 1.38s[0m
[0m2021.07.04 21:43:44 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:43:46 INFO  time: compiled chotay in 1.55s[0m
[0m2021.07.04 21:43:51 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:43:52 INFO  time: compiled chotay in 1.33s[0m
[0m2021.07.04 21:44:17 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:44:19 INFO  time: compiled chotay in 1.97s[0m
[0m2021.07.04 21:44:32 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 18m 56.926s)[0m
[0m2021.07.04 21:44:37 INFO  Listening for transport dt_socket at address: 65353[0m
[0m2021.07.04 21:44:37 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 21:44:37 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 21:44:37 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:44:37 INFO  Trying to attach to remote debuggee VM 127.0.0.1:65353 .[0m
[0m2021.07.04 21:44:37 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 21:44:39 ERROR Sun Jul 04 21:44:39 PKT 2021 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.[0m
[0m2021.07.04 21:44:40 INFO  a. Insert new Customer info[0m
[0m2021.07.04 21:44:40 INFO  b. Search for customer[0m
[0m2021.07.04 21:44:40 INFO  c. Delete Customer record[0m
[0m2021.07.04 21:44:40 INFO  d. Update customer record[0m
[0m2021.07.04 21:44:40 INFO  [0m
[0m2021.07.04 21:44:40 INFO  Enter the Operation to perform (a,b,c.d)[0m
[0m2021.07.04 21:44:57 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 21:53:53 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 21:53:56 INFO  time: compiled chotay in 2.25s[0m
Jul 04, 2021 9:56:03 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1691
Jul 04, 2021 9:57:33 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1749
[0m2021.07.04 22:04:41 INFO  running 'C:\Users\hp\.jabba\jdk\adopt@1.11.0-11\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar C:\Users\hp\AppData\Local\Temp\metals990205431214098239\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'[0m
[0m2021.07.04 22:04:46 INFO  [info] welcome to sbt 1.5.4 (AdoptOpenJDK Java 11.0.11)[0m
[0m2021.07.04 22:04:50 INFO  [info] loading settings for project chotay-build-build from metals.sbt ...[0m
[0m2021.07.04 22:04:53 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project\project[0m
[0m2021.07.04 22:04:56 INFO  [info] loading settings for project chotay-build from metals.sbt ...[0m
[0m2021.07.04 22:04:56 INFO  [info] loading project definition from C:\Users\hp\Desktop\random\chotay\project[0m
[0m2021.07.04 22:05:06 INFO  [success] Generated .bloop\chotay-build.json[0m
[0m2021.07.04 22:05:06 INFO  [success] Total time: 9 s, completed Jul 4, 2021, 10:05:06 PM[0m
[0m2021.07.04 22:05:16 INFO  [info] loading settings for project chotay from build.sbt ...[0m
[0m2021.07.04 22:05:16 INFO  [info] set current project to hello-world (in build file:/C:/Users/hp/Desktop/random/chotay/)[0m
[0m2021.07.04 22:05:45 INFO  [success] Generated .bloop\chotay-test.json[0m
[0m2021.07.04 22:05:45 INFO  [success] Generated .bloop\chotay.json[0m
[0m2021.07.04 22:05:45 INFO  [success] Total time: 29 s, completed Jul 4, 2021, 10:05:46 PM[0m
[0m2021.07.04 22:05:45 INFO  sbt bloopInstall exit: 0[0m
[0m2021.07.04 22:05:46 INFO  time: ran 'sbt bloopInstall' in 1m4s[0m
[0m2021.07.04 22:05:46 INFO  Disconnecting from Bloop session...[0m
[0m2021.07.04 22:05:46 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.04 22:05:46 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.04 22:05:46 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 41220'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Loading 2 projects from 'c:\Users\hp\Desktop\random\chotay\.bloop'...
[0m[32m[D][0m Loading project from 'c:\Users\hp\Desktop\random\chotay\.bloop\chotay.json'
[0m[32m[D][0m Loading project from 'c:\Users\hp\Desktop\random\chotay\.bloop\chotay-test.json'
[0m[32m[D][0m Configured SemanticDB in projects 'chotay', 'chotay-test'
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:41220...
The server is listening for incoming connections at tcp://127.0.0.1:41220...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 22:05:47 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 22:05:47 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 12002'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:12002...
The server is listening for incoming connections at tcp://127.0.0.1:12002...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.04 22:05:47 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.04 22:05:47 INFO  time: Connected to build server in 1.65s[0m
[0m2021.07.04 22:05:47 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.04 22:05:53 INFO  time: indexed workspace in 5.43s[0m
Jul 04, 2021 10:14:16 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1927
[0m2021.07.04 22:14:49 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:14:49 INFO  time: compiled chotay in 0.59s[0m
[0m2021.07.04 22:15:12 INFO  skipping build import with status 'Installed'[0m
[0m2021.07.04 22:15:29 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:15:29 INFO  time: compiled chotay in 0.48s[0m
[0m2021.07.04 22:15:40 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:15:41 INFO  time: compiled chotay in 1.56s[0m
[0m2021.07.04 22:25:29 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:25:31 INFO  time: compiled chotay in 2.02s[0m
[0m2021.07.04 22:29:41 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:29:41 INFO  time: compiled chotay in 0.22s[0m
[0m2021.07.04 22:30:21 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:30:21 INFO  time: compiled chotay in 0.14s[0m
[0m2021.07.04 22:30:35 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:30:35 INFO  time: compiled chotay in 0.11s[0m
[0m2021.07.04 22:30:40 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:30:40 INFO  time: compiled chotay in 0.13s[0m
[0m2021.07.04 22:30:54 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:30:54 INFO  time: compiled chotay in 0.26s[0m
[0m2021.07.04 22:31:02 INFO  C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala:57:9: stale bloop error: rs is already defined as value rs
        val rs = statement.executeQuery("SELECT * FROM customers;")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
[0m2021.07.04 22:31:02 INFO  C:\Users\hp\Desktop\random\chotay\src\main\scala\Main.scala:57:9: stale bloop error: rs is already defined as value rs
        val rs = statement.executeQuery("SELECT * FROM customers;")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
[0m2021.07.04 22:31:08 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:31:08 INFO  time: compiled chotay in 0.21s[0m
[0m2021.07.04 22:37:19 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:37:21 INFO  time: compiled chotay in 1.56s[0m
[0m2021.07.04 22:37:28 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:37:30 INFO  time: compiled chotay in 1.74s[0m
[0m2021.07.04 22:39:04 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:39:04 INFO  time: compiled chotay in 0.21s[0m
[0m2021.07.04 22:40:09 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:40:11 INFO  time: compiled chotay in 1.55s[0m
[0m2021.07.04 22:40:48 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:40:50 INFO  time: compiled chotay in 2.25s[0m
[0m2021.07.04 22:41:58 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:42:00 INFO  time: compiled chotay in 1.92s[0m
[0m2021.07.04 22:42:09 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:42:11 INFO  time: compiled chotay in 1.59s[0m
[0m2021.07.04 22:42:24 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:42:26 INFO  time: compiled chotay in 1.82s[0m
[0m2021.07.04 22:45:28 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:45:28 INFO  time: compiled chotay in 0.23s[0m
[0m2021.07.04 22:45:53 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:45:53 INFO  time: compiled chotay in 0.22s[0m
[0m2021.07.04 22:48:25 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:48:25 INFO  time: compiled chotay in 0.2s[0m
[0m2021.07.04 22:48:45 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:48:45 INFO  time: compiled chotay in 0.22s[0m
[0m2021.07.04 22:49:02 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:49:04 INFO  time: compiled chotay in 1.6s[0m
[0m2021.07.04 22:50:21 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:50:21 INFO  time: compiled chotay in 0.26s[0m
[0m2021.07.04 22:50:57 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:50:57 INFO  time: compiled chotay in 0.15s[0m
[0m2021.07.04 22:51:01 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:51:03 INFO  time: compiled chotay in 1.59s[0m
[0m2021.07.04 22:54:21 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:54:21 INFO  time: compiled chotay in 0.21s[0m
[0m2021.07.04 22:54:34 INFO  compiling chotay (1 scala source)[0m
[0m2021.07.04 22:54:36 INFO  time: compiled chotay in 1.86s[0m
[0m2021.07.04 22:55:16 INFO  Deduplicating compilation of chotay from bsp client 'Metals 0.10.4' (since 49m 29.496s)[0m
[0m2021.07.04 22:55:21 INFO  Listening for transport dt_socket at address: 54620[0m
[0m2021.07.04 22:55:21 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.04 22:55:21 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\hp\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.04 22:55:22 INFO  Starting debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 22:55:22 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54620 .[0m
[0m2021.07.04 22:55:22 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.04 22:55:23 ERROR Sun Jul 04 22:55:23 PKT 2021 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.[0m
[0m2021.07.04 22:55:24 INFO  a. Insert new Customer info[0m
[0m2021.07.04 22:55:24 INFO  b. Search for customer[0m
[0m2021.07.04 22:55:24 INFO  c. Delete Customer record[0m
[0m2021.07.04 22:55:24 INFO  d. Update customer record[0m
[0m2021.07.04 22:55:24 INFO  [0m
[0m2021.07.04 22:55:24 INFO  Enter the Operation to perform (a,b,c.d)[0m
[0m2021.07.04 22:56:07 INFO  Canceling debug proxy for [tests.ScalaJdbcConnectSelect][0m
[0m2021.07.04 22:57:46 INFO  shutting down Metals[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
